### DISCUSSION TITLE
Upgrading From Version 4 to 5.0

### DISCUSSION THREAD

**Discussion by Hamzi-SE** [hakimel/reveal.js#3564] [Q&A]
How can I upgrade my current project from version 4.4 to 5.0? I want to use the new "Scroll View".

Thanks in advance.

**Comment by sebmestrallet** [comment]
Are you still looking for an answer?

How did you get reveal.js in the first place? (git clone, zip download...) Which files have you edited ? (index.html, .css themes, .scss source themes...) Did you add plugins?
